src/Feature/Account/code/Models/Requests/UpdateAccountRequest.cs

```diff
@@ -0,0 +1,35 @@
+﻿//    Copyright 2020 EPAM Systems, Inc.
+// 
+//    Licensed under the Apache License, Version 2.0 (the "License");
+//    you may not use this file except in compliance with the License.
+//    You may obtain a copy of the License at
+// 
+//      http://www.apache.org/licenses/LICENSE-2.0
+// 
+//    Unless required by applicable law or agreed to in writing, software
+//    distributed under the License is distributed on an "AS IS" BASIS,
+//    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//    See the License for the specific language governing permissions and
+//    limitations under the License.
+
+namespace Wooli.Feature.Account.Models.Requests
+{
+    using System.ComponentModel.DataAnnotations;
+    using System.Diagnostics.CodeAnalysis;
+
+    using TypeLite;
+
+    [ExcludeFromCodeCoverage]
+    [TsClass]
+    public class UpdateAccountRequest
+    {
+        [Required]
+        public string ContactId { get; set; }
```

Comment:
Why do we pass ContactId from FE?

Reply:
We get user from manager through `ContactId` so i keep that field in `Request`